test/e2e/specs/reusable-blocks.test.js

```diff
@@ -1,27 +1,21 @@
 /**
  * Internal dependencies
  */
-import '../support/bootstrap';
 import {
 	insertBlock,
-	newDesktopBrowserPage,
 	newPost,
 	pressWithModifier,
 	searchForBlock,
 } from '../support/utils';
 
 function waitForAndAcceptDialog() {
 	return new Promise( ( resolve ) => {
-		page.once( 'dialog', async ( dialog ) => {
-			await dialog.accept();
```

Comment:
Why are we removing this?

Reply:
We have global handler so you don't need to accept it twice.

Reply:
Can we remove the entire `waitForAndAcceptDialog` function, then?

Reply:
We use this `Promise`, so not quite sure 🤷‍♂️